content/rc/databases/active-active-redis.md

```diff
@@ -11,29 +11,29 @@ aliases: [
 
-Active-Active databases are distributed across multiple regions (geo-distribution).  This improves performance by reducing latency for nearby users and improves availability by protecting against data loss in case of network or resource failure.
+Active Active databases are distributed across multiple regions (geo-distribution).  This improves performance by reducing latency for nearby users and improves availability by protecting against data loss in case of network or resource failure.
 
-Active-Active databases allow read and write operations in each copy.  Each copy eventually reflects changes made in other copies ([eventual consistency]({{<relref "/glossary#eventual-consistency">}})).  Conflict-free data types (CRDTs) synchronize read and write operations between copies.  CRDTs ensure consistency and resolve conflicts.
+Active Active databases allow read and write operations in each copy.  Each copy eventually reflects changes made in other copies ([eventual consistency]({{<relref "/glossary#eventual-consistency">}})).  Conflict-free data types (CRDTs) synchronize read and write operations between copies.  CRDTs ensure consistency and resolve conflicts.
 
-## Active-Active geo-distributed replication
-
-Active-Active databases use both clustering and replication to strengthen your database. Active-Active Redis has additional features like geo-distribution, multiple active proxies, conflict resolution, and automatic failover to provide you with a more durable and scalable database.
+## Active Active geo-distributed replication highlights
 
 ### Multi-zone
 
 Geo-distributed replication maintains copies of both primary and replica shards in multiple clusters. These clusters can be spread across multiple availability zones. Active-Active Redis uses zone awareness to spread your primary and replica shards across zones, which helps protect against data loss from regional outages.
 
-### Multiple active proxies
+### Local latency with unique endpoints
 
-Active-Active databases use a multi-primary architecture, which lets you read and write to a primary shard in any of your participating clusters. Having [multiple active proxies]({{<relref "/rs/databases/configure/proxy-policy#about-multiple-active-proxy-support">}}) allows users to connect to the cluster closest to them, reducing latency.
+Applications can specify which copy of the database to connect to allowing for local latency.  This is done by specifying the desired database's unique endpoint in your application configuration.  Each database is a full copy of the data with read and write permissions and will sync data to all database copies in the participating Active Active clusters. 
```

Comment:
Reworded this section a bit. I also think it's okay to remove the last sentence here since this info already appeared in the intro.
```suggestion
Applications can connect to a specific copy of an Active-Active database using its unique endpoint. For local latency, configure your application to use a database endpoint in the closest region.
```